packages/driver/src/cypress/cy.ts

```diff
@@ -195,11 +203,12 @@ export class $Cy extends EventEmitter2 implements ITimeouts, IStability, IAssert
   documentHasFocus: ReturnType<typeof createFocused>['documentHasFocus']
   interceptFocus: ReturnType<typeof createFocused>['interceptFocus']
   interceptBlur: ReturnType<typeof createFocused>['interceptBlur']
+  overrides: IOverrides
 
   private testConfigOverride: TestConfigOverride
   private commandFns: Record<string, Function> = {}
 
-  constructor (specWindow, Cypress, Cookies, state, config) {
+  constructor (specWindow, Cypress, Cookies, state, config, autoRun = true) {
```

Comment:
Can you add notes on autoRun? It's not obvious why/when Cypress wouldn't auto-run.

Reply:
yeah can we also never use booleans as an argument and instead opt for an object literal to avoid stuff like this...

```js
foo(x, y, z, true)

and instead do

foo(x, y, z, { autoRun: true })
```

Reply:
This is vestigial and no longer needed. Removed in https://github.com/cypress-io/cypress/pull/21137/commits/02a63ed840fb3d86153cc14a1fc835ec3212ae25